src/lib/dataAccessS3.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+export const httpPostS3 = (endPoint, body) => {
+    const headers = {
+        "Access-Control-Allow-Headers": "*",
+        "Access-Control-Allow-Origin": "*",
+    };
+
+    return new Promise((resolve, reject) => {
+        axios
+            .post(endPoint, body, { headers: headers, withCredentials: true })
```

Comment:
Ah, withCredentials shouldn't be true here (we are sending this request to S3, and AWS doesn't need to know our credentials 😄

Reply:
Removed :))